cinematch/Screens/Auth/SignUp/Model/InputData.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  InputData.swift
+//  cinematch
+//
+//  Created by Ксения Панкратова on 16.04.2025.
+//
+
+import Foundation
+
+struct InputData {
```

Comment:
Всё что отвечает за обновление данных на экране. не забывай на Hashable подписывать 

```suggestion
struct InputData: Hashable {
```